cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/OkHttpClientInstance.java

```diff
@@ -52,11 +53,12 @@ private OkHttpClientInstance() {
     // package scoped for testing only
     static OkHttpClient createClient() {
         return new OkHttpClient.Builder()
-            .callTimeout(getCallTimeout())
-            .connectTimeout(getConnectTimeout())
-            .readTimeout(getReadTimeout())
-            .writeTimeout(getWriteTimeout())
-            .addInterceptor(LOGGING_INTERCEPTOR)
+                .callTimeout(getCallTimeout())
+                .connectTimeout(getConnectTimeout())
+                .readTimeout(getReadTimeout())
+                .writeTimeout(getWriteTimeout())
+                .addInterceptor(LOGGING_INTERCEPTOR)
+                .cache(CwmsHttpCache.getInstance())
```

Comment:
Oh - and it would be nice if there was a a way to disable or just have the default cache behavior.   Like if some -D was set this method wouldn't call the .cache() method and OkHttp would just work the way it was previously working.